src/Pages/Home.js

```diff
@@ -190,7 +188,7 @@ export default function Home(props) {
C4WhROxu+hOlmQ71eIR/F4Ok3yM1RqWzEi4Uik3DjbwjkjZAJ5iXBBwKcDehOytlQm1fIv3v04ds3YBJiEwhV4EzC1diP+K6n+HKQdz39ZVwSPCAJv/xfhIs4G6gen6qIx7t2fe733wKnUKXndy1mK7AAOpe0w+GYJSzjkuACzcVZs6Heu2HN410/Pvf/OAmFNwG1mS8V5r87+lLsfdETkoQx05JgO0ByJzEbXv2ZDXXdDWueRTGIewisApVJq7noJKBrU2eKWIejlrBMSwLhaFE/G87B5OVO9dsNa8YrQetiCKwcVV4UtiSuDKZnxL2pvYA2WMIyLAktINDNhu/h5OVOtdoNuBJmWQu8VFYt22KyE2ugce1PDNBAkjBhVhJcoNHPhuidsPXYDeMLA4zBHGeAl8qsy5wEvekQ4KnCSktYRiWhCQRzZ0PldwOeGCQUMh0CVdaFwqqZZ4uRWA/+GMj9zCNGJcF2gGYwazZUeDfMGQmFTgdHZdZost3RfDUeAxTfDQN5xAGZlAQfaF4xzAbKVzPzj4R6TAelmhxJSNYAf3k+vhsm20GSMGZQEjygCb8zzIYivpr5/btHjz7dvXu8T0SfDuxN6C74uaez/y4MBnI/83/MSYILROcSs+EJzga6s3wtePM+asFfe/r53AReYTlNODTrIAEfhMQuSBKQMUkgPf6onw03AV/uRPeVqwVRDGL6OV0AXkEJTcAkXF3yqaeBPAg5zZgk7AUC7Wy4DPhypxxesrYA4XKgugjMnDKacCh5gYA/uRTPgTz1NM2QJLSAQj8bCO+EZdsN77EFM/X6Od0EZqqEJqw4McctHA6D0/KIwzRTktAFCv7ZgL6Szg71mJcD/3QIy2jCvCRc/284DORBSGREEppA9IRpNtB3w7Mdm/b0ekpt3779qT4A/MuBfzoEisK18kg+24DHCvcSHzbIU09/mZEEB4gus8wG0m7YsWNP1IIYLAPncihiOjiq8PsTtiz9IORAnnpKMiIJPhDdZJ4N6OuSlwVxmdLwoU/AMB2q1YSdc74u4d6/FuBwkCRMmJAEDwj0s+EnAMNs0O6GNePLgtUqJSyDJg3Dfl5DYBYqio12/iRgCFDiVmZ56ukfA5LgAg3OBnQB8OVOubzGFOgXQq40PO/ndr8aTWhZZKswBgl/Lw7kEYcEA5JAe9hJPxuGEDmdvwhf06SAngZcDnRPgFmgKDqMdzRjHWKfN8gjDjHLPwk+EMybDQ9zxeD7zwen+1vX99RCrb598+ZweLFS04F8nMD/kMP5SQTkXU86yz4JHtDwzoYvX35+/nzk+O2eKtbm1b1JHKowHUBRNGz+JJyNHyzKU0/TlnsSXKAa0mcDhuDrgc+7jj/v4alhSbar7b1RHJ5jHMqYDqGiaOe+o/nq9L0JmAQ0kEcc0DJPgh0A0YUr2WcDhmD/rh/P95RfAu1pQxSH5xiH4qZDwXcsHZr/rqeBPButV50kuK47no2VeNgJ4GeG2YAh2BWFoKfqYLvCOBQ2HRxFsJE9CbdO/0+eeuJhu+42z/Na/oga+0ROwlP1jz/S8jxvm+vapRwkaGfDRYj0ddPgR31CMDsOPc2seAu8CnkAyna3jf9ONrK3J/p713GblsjO7ra1v+XbydQMDb/dtQs+SMDZoHu5UxSCa58/Hz/+vM4dmHPm0Pt3IHkSuIWKoJH+72XHVwwara4lsvFUkbyiDxLgsv6dsLB16+31yy0DM61efZ/7hHHR36eyTXGxRFZu12v7vlos3297Xbf4g4QnM2aDo0wTAjdHpUU5TbC3ef5Glc9oPbiWoLPdES/S8idofw4TLS/ijtjl3ZEAt5OzwcwiKBUAr2K+srk5/ivZ9kcaaf/yeZ7XceUgYeGa7iy63/1q3JEAMNTOBmUk5ibQjhM6HH8h/iP//A1jByxJwNkQKkM5wIr2BihLiOLvSECvki93CpSxmJtAGGBy2idy84Au8bXMr8w8RkDlv+3FEqL4gwQUXw7nTC8C74ECYYRJEkQ5Bwko/Dn9aYOxxwgLGg+ZZ5gkQeTkQ17YhJ8XAlUE/Az2f21/rKFKwz8eHJWFJEHk0wQGN7+MLxGGbxc3GsYJ2Oa6zWwf6nZHmeBORIHjgTIdfEuIHHxg8eTOnScQKG6NKAQ8H4o3x31o4f16ZIWPh2yh9SwhcgA+Duc1QVQC21qM5uT5v4aKq+yFQrbUyk1FIg8XuIRMFwXdAm+Us103aoOaqOyFQqiQ3Kkk4iqbBKdOKdBcN3Sjy4aqXiioDGxLiPKTEJAHQrW+hoM5DU5YeBLkuURRgSSETvYWtLxtbnX/Q2u6nVEaKnOhoFKTrzMRFUiCk2kitL0qXRYswXW9tr+x7CiEco0gCmNDXsGybIFuTxS3HoizzK/r76+okr2QS7C8W5BIg0f5IiIHsiBfhnUsIUp+DDJc8rxgWV7J2u423BMFREGl4VX3ZEbUiwNUoTO7Bd1l2QJCGghRIOyGRkeCIH6zd8eoDQNBFEAhXcC9Gi1s7WYqVz7BLthX0xV81DQKNgZFiSwpQn7vCMPyNTtimbmkGQ/qMfJlw/8R7uaPhhyxUCjcRvPgtJN7GRuRuwnKfgcGL44hTxHzDho/fnTMys6/Z8KtPF4SZMHgK4p45dn0eI8Q+fx+LRlraGr3B+XQ9wU7HR7OHQ7p0qfDlFahDPQGbT4LYhaU4td5EJH1BdM0/V7QepgQCPf9ParPGj7z6DktcX3D2eEimly7MfVxgY9NCoxb9ZyWMMaaWWqHy30IjxXYhpSjluevVas5WEi6trU+pUFtr8rNxjSpJwtWkb6Z1wIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8NUeHBIAAAAACPr/2hU2AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA+AUpJYZRt3AXJgAAAABJRU5ErkJggg=="/>
```

Comment:
@omernadeem1070 this is not ideal it was used before but this way of using images makes code inconsistent can you make svg separate and then import it in Home.js

Reply:
@Yashsharma1911 sorry can you please explain?

Reply:
yes sure in code as you can see currently to use image we are using `img` tag and in that tag `src` attribute contain whole file data which is not ideal it is making code unreadable and longer, `src` attribute should only contain path of image not image itself so request here is to import image in `src` like `./imageName` not `data:image/png...`

Reply:
Got it thanks for the explanation @Yashsharma1911